### PR TITLE
Added Digit_Count.rb

### LINKED ISSUE #1993: Digit-Count -Ruby
Pls assign me this

### PR DESCRIPTION
**Pull request by Harikrishnan6336**
Fixes #1993 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Digit_Count.rb
- 

#### Languages Used:
- Ruby
- 

#### Files Added:
- Digit_Count.rb
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Harikrishnan6336** [comment]
@asha15  Could you please review this

File: Digit_Count/Digit_Count.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#Digit count of given number
+
+puts "Enter the number : "
+num=gets.chomp.to_i
+
+temp=num
+count=0
```
Comment (MastersAbh) [inline Digit_Count/Digit_Count.rb]: space around all operators

File: Digit_Count/Digit_Count.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#Loop to determine the number of digits
+while (temp>0)
+    count+=1
+    temp=temp/10
+end
```
Comment (MastersAbh) [inline Digit_Count/Digit_Count.rb]: space around operators

File: Digit_Count/Digit_Count.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+puts "The given number #{num} has #{count} digits in it"
+
+#SAMPLE INPUT AND OUTPUT
```
Comment (MastersAbh) [inline Digit_Count/Digit_Count.rb]: please add a case for 0 as well. 
Your code will say 0 has 0 digits. 
Also, can you modify to add case for -ve numbers ?

**Comment by Harikrishnan6336** [comment]
@MastersAbh  I have done the changes

**Pull request merged**